static/app/types/user.tsx

```diff
@@ -46,11 +46,11 @@ export interface User extends Omit<AvatarUser, 'options'> {
     avatarType: Avatar['avatarType'];
     clock24Hours: boolean;
     defaultIssueEvent: 'recommended' | 'latest' | 'oldest';
-    issueDetailsNewExperienceQ42023: boolean;
     language: string;
     stacktraceOrder: number;
     theme: 'system' | 'light' | 'dark';
     timezone: string;
+    prefersIssueDetailsStreamlinedUI?: boolean;
```

Comment:
is there a reason why it's optional? won't we return 'false' in all the cases where it isn't set